Bottom: fa85202bff648ab2dfea58006b458f519240e26c
Top:    b9491dbee3714d904925acb7b69bf5a4bdbf6bb0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-12 01:31:20 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index a787e87..d38d720 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
